tools/infrastructure/docker/AGL/workspace/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ubuntu:16.04
```

Comment:
@mkorniichuk I would avoid committing workplace for AGL in repo

Reply:
@LuxoftAKutsan fixed within 472c3a9a75a1670991f453d0e6453cb371e03e3b